### ISSUE TITLE
install: Installing via a GitHub branch that has a forward slash in its name gives invalid summary information

### ISSUE THREAD

**Issue by HenrikBengtsson** [callr-org/website#6]
When installing a branch from GitHub that contains a forward slash (`/`) in the branch name, e.g. `feature/binMeans-weights`, the callr.org/install script reports (incorrectly) that the installation failed at the end.  The summary table is also incorrect.  Note that it is only the information that is wrong here.  The actual installation does indeed succeed.

Example:

``` r
> source("http://callr.org/install#HenrikBengtsson/matrixStats@feature/binMeans-weights")
Installing/updating package: 'binMeans-weights'
Installing packages from GitHub:
Downloading github repo HenrikBengtsson/matrixStats@feature/binMeans-weights
Installing matrixStats
'/Library/Frameworks/R.framework/Resources/bin/R' --vanilla CMD INSTALL  \
  '/private/tmp/RtmpvF8IYs/devtoolsd80a1d0c0a23/HenrikBengtsson-matrixStats-9e8c27b'  \
  --library='/Users/hb/Library/R/3.2/library' --install-tests

* installing *source* package 'matrixStats' ...
[...]
Package updates:
                 Previous Current
binMeans-weights     <NA>    <NA>

Failed to install requested package: 'binMeans-weights'
> matrixStats::binMeans
```

**Comment by HenrikBengtsson** [comment]
Another related case reported by @pneuvial:

``` r
> source("http://callr.org/install#http://braju.com/xfer/pn/async_0.1.3.tar.gz")
[...]
Package updates:
                  Previous Current
async                  <NA>   0.1.3
async_0.1.3.tar.gz     <NA>    <NA>

Failed to install requested package: 'async_0.1.3.tar.gz'
```

**Comment by HenrikBengtsson** [comment]
This was solved as part of Issue #11.
